core/frontend/src/test/BlankConnection.test.ts

```diff
@@ -1,10 +1,11 @@
 /*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
-import { afterAll, beforeAll, describe, expect, it } from "vitest";
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { afterAll, beforeAll, describe, expect, it, vi } from "vitest";
```

Comment:
I think we should keep a space between copyright comment and first import statement.

```suggestion
 *--------------------------------------------------------------------------------------------*/
 
import { afterAll, beforeAll, describe, expect, it, vi } from "vitest";
```

Reply:
Core packages don't have this newline in their source files, so I'd rather not add it for consistency.